### ISSUE TITLE
Add Swift support

### ISSUE THREAD

**Issue by vincent-psarga** [hiptest/hiptest-publisher#20]

**Comment by wills-dtb** [comment]
Any ETA on this?

**Comment by tvial-tagheuer** [comment]
It seems there's a PR here: https://github.com/hiptest/hiptest-publisher/pull/134

**Comment by vincent-psarga** [comment]
yes indeed, the main problem with this PR right now is that I've been unable to make a proper `hps-swift` example to validate the generated code.

For all languages/framework supported by hiptest-publisher, we have our own implementation of the coffee machine sample (generally named `hps-<language>-<framework>`).

**Comment by tvial-tagheuer** [comment]
Maybe @bangroot can help on this? It will really help the community.

**Comment by bangroot** [comment]
Oh, I didn’t realize that was a need. I’ll see what we can do. Might be a bit :/.

**Comment by vincent-psarga** [comment]
I merged your PR anyway, it should not have waited that long, sorry about that.

Maybe we should have some explanations about that in the contributing MD file. But so far, when people contributed to other languages, it was pretty straightforward to get an example up and running, so I did not consider asking contributors to also add the `hps-`.

Once again, sorry for taking too long to get this PR merged. I'll try to do a release of the publisher asap :)
